File: frontend/src/modules/assemblies/manifest.ts
// DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
// Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
import { Layers } from 'lucide-react';
import type { ModuleManifest } from '../_types';

// module registry lineage: ddc-lineage:a17f93c4-assemblies-01
export const manifest: ModuleManifest = {
  id: 'assemblies',
  name: 'nav.assemblies',
  description: 'modules.assemblies_desc',
  version: '1.0.0',
  icon: Layers,
  category: 'estimation',
  defaultEnabled: true,
  routes: [],
  navItems: [],
};

File: backend/app/modules/cost_match/manifest.py
# DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
# Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
"""Cost-match module manifest."""

from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="oe_cost_match",
    version="0.1.0",
    display_name="Cost Match",
    description=(
        "Automatic CWICR item matching for material layers: exact "
        "match first, then semantic via Qdrant when [semantic] extra is "
        "installed, then needs-review candidates."
    ),
    author="OpenConstructionERP Core Team",
    category="core",
    depends=["oe_users", "oe_projects", "oe_costs", "oe_dashboards"],
    optional_depends=["oe_ai"],
    auto_install=True,
    enabled=True,
)
